enterprise/backup/src/test/java/org/neo4j/backup/BackupToolIT.java

```diff
@@ -70,6 +71,7 @@ public void setUp() throws Exception
     public void teardown() throws Exception
     {
         pageCache.close();
+        fs.close();
```

Comment:
Lets keep that away from this PR there are too much places that will eventually require almost full rewrite as soon as we will be more strict with FS abstraction